Resource generator C++ source includes redundant semicolon

It generates warnings like this:
```
> Task :sysid-projects:mechanism:compileEmbeddedBinaryReleaseStaticLibraryEmbeddedBinaryCpp
Warnings in file frcUserProgram.cpp ....
/home/runner/work/sysid/sysid/sysid-projects/mechanism/build/generated/exe/frcUserProgram.cpp:8:2: warning: extra ‘;’ [-Wpedantic]
    8 | };
      |  ^
```
It's probably this line:
https://github.com/wpilibsuite/native-utils/blob/main/src/main/java/edu/wpi/first/nativeutils/resources/ResourceGenerationAction.java#L54

--- a/build.gradle
+++ b/build.gradle
@@ -12,7 +12,7 @@ targetCompatibility = 11
 
 allprojects {
     group = "edu.wpi.first"
-    version = "2022.8.1"
+    version = "2022.8.2"
 
     if (project.hasProperty('publishVersion')) {
         version = project.publishVersion

--- a/src/main/java/edu/wpi/first/nativeutils/resources/ResourceGenerationAction.java
+++ b/src/main/java/edu/wpi/first/nativeutils/resources/ResourceGenerationAction.java
@@ -51,7 +51,7 @@ public abstract class ResourceGenerationAction implements WorkAction<ResourceWor
 
             writer.append("  *len = ");
             writer.append(Integer.toString(sizeCount));
-            writer.append(";\n  return contents;\n};\n");
+            writer.append(";\n  return contents;\n}\n");
             writer.append("}  // extern \"C\"\n");
 
             if (namespace != null && !namespace.isEmpty()) {

